app/lib/logic/provider_signup.rb

```diff
@@ -51,6 +51,7 @@ def signup_provider(application_plan = nil, options = {})
         #
         self.class.transaction do
           provider.save!
+          provider.update!(first_admin_id: user.id)
```

Comment:
We do not need that anymore right?
Everything is handled by signup result

Reply:
I don't think so 🤔 If it was, we could remove all this code, but I think there still one scenario that goes through here that I right now I don't remember which one 🤔 but I rather not taking unnecessary risks.